gitlab/const.py

```diff
@@ -17,6 +17,42 @@
 
 from gitlab.__version__ import __title__, __version__
 
+# NOTE(jlvillal): '__deprecated_all__' (our own custom name, not a Python
+# standard name) only affects users accessing constants via the top-level
+# namespace. See 'gitlab/__init__.py:__getattr__()' for the consumer of
+# '__deprecated_all__' For example 'x = gitlab.NO_ACCESS'.  We want users to
+# instead use constants by doing code like: gitlab.const.NO_ACCESS.
```

Comment:
I'm sure a lot of people who mainly do scripting will always do `from gitlab.const import ...` into their own modules and access them directly, but agreed about polluting the top `gitlab.` module access. So I think the explicit namespace like that in examples makes sense but not everyone will do it :grinning:

Reply:
I'm thinking nothing for me to do in regards to this comment. But re-open if I am wrong.